F50/Platform/Platform.shared.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Xamarin.F50
+{
+	public partial class Platform

```

Comment:
I have a question about the name. Right now the namespace is F50, but what will this be in future. Will we ever separate the objects with namespaces? `Xamarin.F50.Preferences`? If we do, then the class name and namespace will clash.

Reply:
We thought about this, but to be honest just easier to have 1 name space. If we have a clash that means we did something wrong for sure.